packages/core/admin/admin/src/features/Auth.tsx

```diff
@@ -154,10 +155,11 @@ const AuthProvider = ({ children }: AuthProviderProps) => {
   );
 
   const logout = React.useCallback(async () => {
+    dispatch(adminApi.util.resetApiState());
     await logoutMutation();
     clearStorage();
     push('/auth/login');
-  }, [clearStorage, logoutMutation, push]);
+  }, [clearStorage, logoutMutation, push, dispatch]);
```

Comment:
fun fact, if resetApiState isn't the first thing in that function, it triggers a window reload on the login page. Not sure why